src/vnsw/agent/test/test_pkt0_interface.h

```diff
@@ -81,7 +81,7 @@ class TestPkt0Interface : public VrouterControlInterface {
         memset((char *) &sin, 0, sizeof(sin));
         sin.sin_family = AF_INET;
         sin.sin_addr.s_addr = inet_addr("127.0.0.1");
-        if (bind(pkt0_client_fd_, (sockaddr *) &sin, sizeof(sin)) == -1) {
+        if (::bind(pkt0_client_fd_, (sockaddr *) &sin, sizeof(sin)) == -1) {
```

Comment:
there is a bind function in the boost library. The compiler complained of the conflict IIRC.